test/e2e/tests/smart-transactions/smart-transactions.spec.ts

```diff
@@ -33,6 +33,7 @@ async function withFixturesForSmartTransactions(
       title,
       localNodeOptions: {
         hardfork: 'london',
+        chainId: '1',
```

Comment:
:information_source:  in this spec we are using mainnet as the selected network. If we don't pass chainid 1 in anvil, the chainId will be incorrect (1337) and the gas mock won't be apply in the global mocks, as we rely on {chainId}